decoder/vaapidecoder_h264.cpp

```diff
@@ -1238,7 +1238,9 @@ YamiStatus VaapiDecoderH264::decodePps(NalUnit* nalu)
 {
     SharedPtr<PPS> pps(new PPS());
 
-    memset(pps.get(), 0, sizeof(PPS));
+    //memset(pps.get(), 0, sizeof(PPS));
```

Comment:
if you memset things in constructor, you do not need this line, it already set by SharedPtr<PPS> pps(new PPS());

Reply:
please remove comments